lizard_auth_server/models.py

```diff
@@ -137,6 +137,9 @@ class UserProfile(models.Model):
     updated_at = models.DateTimeField(auto_now=True, editable=False)
     organisations = models.ManyToManyField(
         "Organisation", blank=True, null=True)
+
+    billable_organisation = models.ForeignKey(Organisation, null=True)
```

Comment:
Dit veld mag null zijn in de DB, maar het is wel nog steeds een verplicht veld. De meeste sites hebben niets met billable organisaties te maken => graag `blank=True` toevoegen.